helm/vitess/templates/_orchestrator-conf.tpl

```diff
@@ -0,0 +1,163 @@
+###################################
+# Orchestrator Config
+###################################
+{{- define "orchestrator-config" -}}
+# set tuple values to more recognizable variables
+{{- $orc := index . 0 -}}
+{{- $namespace := index . 1 -}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: orchestrator-cm
+data: 
+  orchestrator.conf.json: |-
+    {
+    "AccessTokenExpiryMinutes": 1440,
+    "AccessTokenUseExpirySeconds": 60,
+    "ActiveNodeExpireSeconds": 5,
+    "AgentPollMinutes": 60,
+    "AgentsServerPort": ":3001",
+    "AgentSSLCAFile": "",
+    "AgentSSLCertFile": "",
+    "AgentSSLPrivateKeyFile": "",
+    "AgentSSLSkipVerify": false,
+    "AgentSSLValidOUs": [
+    ],
+    "AgentsUseMutualTLS": false,
+    "AgentsUseSSL": false,
+    "ApplyMySQLPromotionAfterMasterFailover": true,
+    "AuditLogFile": "/tmp/orchestrator-audit.log",
+    "AuditToSyslog": false,
+    "AuthenticationMethod": "",
+    "AuthUserHeader": "",
+    "AutoPseudoGTID": false,
+    "BackendDB": "sqlite",
+    "BinlogEventsChunkSize": 10000,
+    "CandidateInstanceExpireMinutes": 60,
+    "ClusterNameToAlias": {
+        "127.0.0.1": "test suite"
+    },
+    "CoMasterRecoveryMustPromoteOtherCoMaster": true,
+    "DataCenterPattern": "[.]([^.]+)[.][^.]+[.]mydomain[.]com",
+    "Debug": true,
+    "DefaultInstancePort": 3306,
+    "DefaultRaftPort": 10008,
+    "DetachLostSlavesAfterMasterFailover": true,
+    "DetectClusterAliasQuery": "SELECT value FROM _vt.local_metadata WHERE name='ClusterAlias'",
+    "DetectClusterDomainQuery": "",
+    "DetectInstanceAliasQuery": "SELECT value FROM _vt.local_metadata WHERE name='Alias'",
+    "DetectPromotionRuleQuery": "SELECT value FROM _vt.local_metadata WHERE name='PromotionRule'",
+    "DetectPseudoGTIDQuery": "",
+    "DetectSemiSyncEnforcedQuery": "SELECT @@global.rpl_semi_sync_master_wait_no_slave AND @@global.rpl_semi_sync_master_timeout > 1000000",
+    "DiscoverByShowSlaveHosts": true,
```

Comment:
only if `report_host` and `report_port`are  enabled on all servers.

Reply:
@enisoc Do you know about `report_host` and `report_port`?

Reply:
I previously set `report-host` in `templates/_vttablet.tpl` so that orc uses the headless service to resolve addresses. Before that, I think MySQL was picking some hostname on its own that isn't resolvable from outside the Pod. I left `report-port` unspecified so it defaulted to the bind port.

Reply:
Ok, I forgot about that, I'm still setting it. Sounds like we should be good to go already for this.